src/app/dynamic/enterprise/metering/component.ts

```diff
@@ -49,9 +49,9 @@ export class MeteringComponent implements OnInit {
       .pipe(map(seeds => (seeds.length > 0 ? seeds[0] : null)))
       .pipe(filter(seed => seed !== null))
       .pipe(switchMap(seed => this._dcService.seedSettings(seed)))
-      .pipe(map(settings => settings.metering))
+      .pipe(pluck('metering'))
```

Comment:
Even though `pluck` requires a couple of characters less than the `map`, we are losing type safety and checking completely due to direct string reference of the field name. I'd rather use `map` as it allows a compile-time check of the field reference for the metering object.